### ISSUE TITLE
Cleanup `models.yml`

### ISSUE THREAD

**Issue by jsangmeister** [OpenSlides/openslides-backend#1648]
We should introduce a formatter for the `models.yml`. Currently, the order of the attributes of the fields is non-coherent, as well as whether certain attributes are set or not in the default case.

**Comment by jsangmeister** [comment]
We need to define a common order of attributes for this in which they should appear for all fields.

**Comment by r-peschke** [comment]
> We should introduce a formatter for the `models.yml`. Currently, the order of the attributes of the fields is non-coherent, as well as whether certain attributes are set or not in the default case.

Yes, but we should wait with re-organization until we have a stabel models.yml again. For now it's better we do not change the sequence, because it will be fairly more complicated to compare a model.yml with and without template fields, because we changed a lot and renamed a lot of fields.

**Comment by jsangmeister** [comment]
I agree, we should of course wait until the template field removal is done.

**Comment by rrenkert** [comment]
Will be done in https://github.com/OpenSlides/openslides-meta/pull/487
